capability/capability_test.go

```diff
@@ -111,49 +112,42 @@ func TestAmbientCapSet(t *testing.T) {
 	}
 	requirePCapSet(t)
 
-	capBounding := []Cap{CAP_KILL, CAP_CHOWN, CAP_SYSLOG}
-	capPermitted := []Cap{CAP_KILL, CAP_CHOWN}
-	capEffective := []Cap{CAP_KILL}
-	capInheritable := []Cap{CAP_KILL, CAP_CHOWN}
-	capAmbient := []Cap{CAP_KILL, CAP_CHOWN}
-
 	pid, err := NewPid2(0)
 	if err != nil {
 		t.Fatal(err)
 	}
-	pid.Set(BOUNDING, capBounding...)
-	pid.Set(PERMITTED, capPermitted...)
-	pid.Set(EFFECTIVE, capEffective...)
-	pid.Set(INHERITABLE, capInheritable...)
-	pid.Set(AMBIENT, capAmbient...)
-	if err = pid.Apply(CAPS | BOUNDING | AMBIENT); err != nil {
+
+	list := []Cap{CAP_KILL, CAP_CHOWN, CAP_SYS_CHROOT}
```

Comment:
From docker to kubernetes, they always fill each type cap sets with the same cap bits, I think maybe it's wrong. So I want to test cap sets with different caps, this is my thought.
But anyway, it's not worth to be different in this test.

Reply:
Yes, I understand that (what I don't understand is why they do it that way -- for example, why drop bounding capabilities at all; but that is way out of scope for this PR).